src/siddi.ts

```diff
@@ -85,6 +88,18 @@ export class Siddi {
           }
         }
 
+        // Exclude sending event parameters for particular event
+        // when those defined in denyParameters config
+        if(eventProperties && config.denyParameters) {
+          config.denyParameters.some(function(element: any, index: number) {
```

Comment:
We can optimize this further but this should okay for now, since there is few params